scripts/publish-functions.sh

```diff
@@ -44,10 +44,21 @@ s3_cp() {
 	if [[ ! -z "${AWS_PROFILE}" ]]; then
 		profile="--profile=${AWS_PROFILE}"
 	fi
+	if [[ ! -z "${AWS_PROFILE_PUSH}" ]]; then
+		profile_push="--profile=${AWS_PROFILE_PUSH}"
+	else
+		profile_push=$profile
+	fi
 	acl="public-read"
 	if [[ ! -z "${S3_ACL_OVERRIDE}" ]]; then
 		acl="${S3_ACL_OVERRIDE}"
 	fi
 	echo "Copying ${1} to ${2}"
-	aws ${profile} s3 cp "${1}" "${2}" "--acl=${acl}"
+	# for partitioned regions we have to push then pull
```

Comment:
Can you modify this so that it's more clear? `for partitioned regions, we cannot copy files between buckets directly. Save it locally and copy it over`. Alternatively, can the staging script be modified so that staging artifacts are pushed to all partitions?